serverless.yml

```diff
@@ -39,6 +39,50 @@ functions:
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
     vpc: ${self:custom.vpc}
 
+resources:
+  Resources:
+    snsTopic:
+      Type: AWS::SNS::Topic
```

Comment:
Not sure if we want each individual lambda to have its own topic or if we want one "uber". I'm currently thinking individual topics....